docs/docsite/rst/become.rst

```diff
@@ -63,23 +66,23 @@ ansible_become
     equivalent of the become directive, decides if privilege escalation is used or not.
 
 ansible_become_method
-    allows to set privilege escalation method
+    which privilege escalation method should be used
 
 ansible_become_user
-    allows to set the user you become through privilege escalation, does not imply `ansible_become: True`
+    set the user you become through privilege escalation, does not imply ``ansible_become: yes``
 
 ansible_become_pass
-    allows you to set the privilege escalation password
+    set the privilege escalation password. See :doc:`playbooks_vault` for details on how to avoid having secrets in plain text
```

Comment:
Change the comma to a semicolon or a new sentence

Reply:
Where?